proposals/0071-Entity/01-context.md

```diff
@@ -0,0 +1,455 @@
+
+This conflation forces users to rely on verbose `group_left` joins to attach metadata to metrics, creates storage inefficiency for constant values, and loses the semantic distinction between what identifies an entity and what describes it.
+
+By introducing Entities as a native concept, Prometheus can provide cleaner query ergonomics, optimized storage for metadata, explicit lifecycle management, and proper semantics that distinguish between identifying labels (what makes an entity unique) and descriptive labels (additional context about that entity).
+
+This proposal also aligns with Prometheus's commitment to being the default store for OpenTelemetry metrics. OpenTelemetry's Entity model provides a well-defined structure for representing monitored objects, and native Entity support enables seamless translation between OTel Entities and Prometheus.
+
+---
+
+## Terminology
+
+Before diving into the problem and proposed solution, let's establish a shared vocabulary:
+
+#### Info Metric
+
+A metric that exposes metadata about a monitored entity rather than a measurement. In current Prometheus convention, these are gauges with a constant value of `1` and labels containing the metadata. Examples include `node_uname_info`, `kube_pod_info`, and `target_info`.
+
+```
+build_info{version="1.2.3", revision="abc123", goversion="go1.21"} 1
+```
+
+#### Entity
+
+An **Entity** represents a distinct object of interest that produces or is associated with telemetry. Unlike Info metrics, Entities are not metrics—they are first-class objects with their own identity, labels, and lifecycle.
+
+In OpenTelemetry, an entity is an object of interest that produces telemetry data. Entities represent things like services, hosts, containers, or Kubernetes pods. Each entity has a type (e.g., `k8s.pod`, `host`, `service`) and a set of attributes that describe it.
+
+This proposal adopts the Entity concept as the native Prometheus representation for what was previously expressed through Info metric conventions.
+
+#### Resource Attributes
+
+In OpenTelemetry, **resource attributes** are key-value pairs that describe the entity producing telemetry. These attributes are attached to all telemetry (metrics, logs, traces) from that entity. When OTel metrics are exported to Prometheus, resource attributes typically become labels on a `target_info` metric.
+
+#### Identifying Labels
+
+**Identifying labels** uniquely distinguish one entity from another of the same type. These labels:
+- Must remain constant for the lifetime of the entity
+- Together form a unique identifier for the entity
+- Are required to identify which entity produced the telemetry
+
+Examples:
+- `k8s.pod.uid` or (`k8s.pod.name`,`k8s.namespace.name`) for a Kubernetes pod
+- `host.id` for a host
+- `service.instance.id` for a service instance
+
+#### Descriptive Labels
+
+**Descriptive labels** provide additional context about an entity but do not serve to uniquely identify it. These labels:
```

Comment:
As described regarding "Identifying Labels", I fear that we are here simply duplicating OTel Entity Data Model design. It feels unnecessary.